rca/utils/embed_finders.py

```diff
@@ -20,3 +24,27 @@ def find_embed(self, url, max_width=None):
             embed["html"] = str(soup)
 
         return embed
+
+
+class WixEmbedFinder(EmbedFinder):
+    """Embed finder support for Wix. Wix does not have oEmbed support."""
+
+    def accept(self, url):
+        return re.match(r"^https?://(?:www\.)?embed.wix\.com/.+$", url)
+
+    def find_embed(self, url, max_width=None):
+        # Attempt to fetch the page content and scrape the title.
+        response = requests.get(url)
+        soup = BeautifulSoup(response.content, "html.parser")
+
+        title_tag = soup.find("title")
+        title = title_tag.string if title_tag else "No title found"
```

Comment:
As per intranet pr:

> Is "No title found" the right thing to return here? Wouldn't none be better to avoid rendering it in the template?

Reply:
Makes sense! I've made it `None` now. Might make it confusing too if the title of the embed is "No title found".